lib/service/service.go

```diff
@@ -309,6 +311,11 @@ func (process *TeleportProcess) GetBackend() backend.Backend {
 	return process.backend
 }
 
+// GetClock returns the process' clock
+func (process *TeleportProcess) GetClock() clockwork.Clock {
```

Comment:
Does this need a getter, or can the clock just be an exported (but not embedded) field?

Reply:
No, no specific need for the getter - I'll update.